core/iwasm/interpreter/wasm_loader.c

```diff
@@ -12906,12 +12939,9 @@ wasm_loader_prepare_bytecode(WASMModule *module, WASMFunction *func,
                     uint8 opcode_tmp = WASM_OP_SELECT;
 
                     if (type == VALUE_TYPE_V128) {
-#if (WASM_ENABLE_SIMD == 0)                                        \
-    || ((WASM_ENABLE_WAMR_COMPILER == 0) && (WASM_ENABLE_JIT == 0) \
-        && (WASM_ENABLE_FAST_INTERP == 0))
-                        set_error_buf(error_buf, error_buf_size,
-                                      "SIMD v128 type isn't supported");
-                        goto fail;
+#if WASM_ENABLE_JIT != 0 \
```

Comment:
Still need to output information like *SIMD v128 type isn't supported* for classic-interp.